Lib/blackrenderer/render.py

```diff
@@ -97,11 +100,11 @@ def buildGlyphLine(infos, positions, glyphNames):
     return glyphLine
 
 
-def calcGlyphLineBounds(glyphLine, font):
+def calcGlyphLineBounds(glyphLine, font, useFontMetrics):
     bounds = None
+    glyphLineBounds = [font.getGlyphBounds(glyph.name) for glyph in glyphLine]
```

Comment:
Why this (and the zip below) change? It doesn't seem to do anything for your patch.

Reply:
I mean, why not. It made it easier to try things out while patching.

Reply:
Than now is the time to change it back, pretty please. It's a change that is irrelvant for your patch, therefore it's noise.